### DISCUSSION TITLE
formula to calculate latitudeDelta and longitudeDelta from coords

### DISCUSSION THREAD

**Discussion by crisfcodes** [react-native-maps/react-native-maps#4533] [Q&A]
I'm using react-native-maps library and I want to calculate longitudeDelta and latitudeDelta with a given coords(longitude, latitude).

I've done a couple of [google](https://stackoverflow.com/a/36688156) [searches](https://pusher.com/tutorials/carpooling-react-native-part-2/) but the formulas I find don't apply to react-native or the answers on how to perform these calculations are not clear. I need to calculate those deltas according to the screen size of the user's device

**Comment by jduhking** [comment]
Any updates on this? I'm curious because I am using expo location with react native maps and expo location only provides latitude and longitude but not deltas
